modules/builds-strategy-custom-build.adoc

```diff
@@ -10,7 +10,9 @@ responsible for the entire build process. Using your own builder image allows
 you to customize your build process.
 
 A Custom builder image is a plain Docker-formatted container image embedded with
-build process logic, for example for building RPMs or base images. The
-`openshift/origin-custom-docker-builder` image is available on the
-https://registry.hub.docker.com/u/openshift/origin-custom-docker-builder[Docker
-Hub] registry as an example implementation of a Custom builder image.
```

Comment:
this image relies on the docker socket so it's not worth pointing to anymore.  The tutorial i just put together is better anyway.